packages/integration-sdk-runtime/src/execution/executeIntegration.ts

```diff
@@ -243,6 +252,8 @@ export async function executeWithContext<
       createStepGraphObjectDataUploader,
       beforeAddEntity: config.beforeAddEntity,
       beforeAddRelationship: config.beforeAddRelationship,
+      afterAddEntity: createAfterAddEntityInternalHook(logger),
```

Comment:
Do we think we may ever want to allow user to override these hooks? Could be added in future change, but we could export the default hook creation function and allow people to slap extra logic onto their hook beyond metric publishing.

Reply:
Right. My thinking is the same. We can easily add in the future, but I don't want to add a preemptively add an unused feature that may never be used.